src/pantab/_reader.py

```diff
@@ -7,26 +7,54 @@
 import pantab.libpantab as libpantab
 
 
+class PantabStream:
```

Comment:
@Priyansh121096 do you know how this will show up in the docs? For now I might be fine to exclude it; this class is not really meant to be used directly by users, but rather is something you would forward along to other classes (like pyarrow's `RecordBatchReader.from_stream` constructor)

Reply:
Ah when building locally I see it doesn't show up. I guess it must have to be in the top level API to appear. Cool!

Reply:
Yes, if this is needed in the docs, I think we'll need to add a directive like below to `doc/source/api.rst` or export `PantabStream` from `pantab/__init__.py`.
```
.. automodule:: pantab._reader
   :members:
```